src/Build.UnitTests/Evaluation/ImportFromMSBuildExtensionsPath_Tests.cs

```diff
@@ -52,7 +47,11 @@ public void ImportFromExtensionsPathNotFound()
                 extnDir1 = GetNewExtensionsPathAndCreateFile("extensions1", Path.Combine("foo", "extn.proj"), GetExtensionTargetsFileContent1());
                 mainProjectPath = ObjectModelHelpers.CreateFileInTempProjectDirectory("main.proj", GetMainTargetFileContent());
 
+#if FEATURE_SYSTEM_CONFIGURATION
                 var projColln = new ProjectCollection();
+#else
+                var projColln = new ProjectCollection(ToolsetDefinitionLocations.ConfigurationFile);
+#endif
```

Comment:
Would it be possible to extract this to a new method `GetProjectCollection()` to avoid repeating this code in multiple places?

Reply:
Oops forgot to reply to this one but this is a good call I made this change as well 😃